standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestObjectStore.java

```diff
@@ -643,14 +644,20 @@ public void testDirectSQLDropPartitionsCacheInSession()
     createPartitionedTable(false, false);
     // query the partitions with JDO
     List<Partition> partitions;
-    try(AutoCloseable c =deadline()) {
+    try(AutoCloseable c = deadline()) {
       partitions = objectStore.getPartitionsInternal(DEFAULT_CATALOG_NAME, DB1, TABLE1,
           10, false, true);
     }
     Assert.assertEquals(3, partitions.size());
 
+    try(AutoCloseable c = deadline()) {
+      List<List<ColumnStatistics>> colStats = objectStore.getPartitionColumnStatistics(DEFAULT_CATALOG_NAME, DB1, TABLE1,
```

Comment:
Add this query to avoid `PART_COL_STATS` table not found exception in direct SQL drop partitions. Not sure why `PART_COL_STATS` table will not be created without this pre-query.